.circleci/infrastructure/workflows.yml

```diff
@@ -0,0 +1,17 @@
+#workflows:
+  enable-versioning-dev:
+    jobs:
+      - enable-versioning
+
+  enable-versioning-production:
+    unless: << pipeline.parameters.run_dev_deployment >>
+    jobs:
+      - enable-versioning:
+          target_env: prod
+          cf-password: CF_PASSWORD_PROD
+          cf-username: CF_USERNAME_PROD
+          cf-space: tanf-prod
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
Ultimately, s3 file versioning should be enabled via terraform and this circle ci workflow should be removed.

Reply:
@jtimpe @n0remac can you confirm that this is what will ensure this works in prod? and is this set up for staging?

Reply:
> @jtimpe @n0remac can you confirm that this is what will ensure this works in prod? and is this set up for staging?

This will now work with the latest commits from @George-Hudson .